plugin.py

```diff
@@ -1,30 +1,18 @@
 from LSP.plugin import AbstractPlugin
+from LSP.plugin import ClientConfig
 from LSP.plugin import register_plugin
 from LSP.plugin import unregister_plugin
-from LSP.plugin.core.typing import List, Optional
 from LSP.plugin import WorkspaceFolder
-from LSP.plugin import ClientConfig
-from LSP.plugin.core.logging import debug
-import sublime
-from os.path import realpath
+from LSP.plugin.core.typing import List, Optional
 import shutil
+import sublime
 
 
-name = "rust-analyzer"
-
-def which_realpath(exe: str) -> Optional[str]:
-    path = shutil.which(exe)
-    if path:
-        return realpath(path)
-    return None
-
```

Comment:
Remove this function.

I'm not sure what's the point of using `realpath`. Can you explain?

My thinking is that is `shutil` finds the path then we will be able to start it. Are there cases when it's not true?

Reply:
Yeah thats fair